en/developer_guide/routing.rst

```diff
@@ -69,28 +69,40 @@ it will be redirect to the ``http://www.open-orchestra.com`` website.
 If a user come on ``http://example.fr/fr/open-orchestra`` so the french site with the good pattern,
 it will be redirect to the ``HOME`` node of the french page.
 
-But if a user come on ``http://example.fr/fr/oo``, it will have a 404 page not found error because redirections are attached to languages.
+But if a user come on ``http://example.fr/fr/oo``, it will have a 404 page not found error because redirections
+are attached to languages.
 
-Implementation
---------------
+Implementations
+---------------
 
-``open-orchestra-front-bundle`` contains all the routing related classes in the ``routing`` folder.
+Open Orchestra provides two alternatives mechanism for the routing :
 
+* Get the url directly from the database (default configuration)
+* Dump the route in a cache file
 
-Route Loader
-~~~~~~~~~~~~
+We are going to see how to configure one or the other, then which solution you could choose for your project.
 
-In Open Orchestra, route collections are extracted from the nodes by some classes:
+Routing from the database
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ``DatabaseRouteLoader`` generates routes for different nodes.
-Routes generated are linked to the ``NodeController`` and have languages, node, site and alias as parameters.
-Route name is generated by two id : ``AliasID_NodeId``.
+In the database, Open Orchestra saves all the routes (for the nodes, and redirections) with document implementing
+the ``RouteDocumentInterface``.
```

Comment:
This part is a good start but I think it needs a little more details: you should introduce that we're talking about front routes, which mainly are node access. This part does not cover the Back Office routing. Maybe you could introduce the routers chain too which is responsible for the Symfony routes.